### ISSUE TITLE
[Blackhole bringup] Missing ckernel SFPU files

### ISSUE THREAD

**Issue by abhullar-tt** [tenstorrent/tt-metal#10158]
Repro steps for softplus
```
./build_metal.sh
./create_env
source python_env/bin/activate
pytest tests/ttnn/unit_tests/operations/test_activation.py::test_mish
pytest tests/ttnn/unit_tests/operations/test_activation.py::test_softplus
```

**Comment by rtawfik01** [comment]
Can you rename this issue "Missing ckernel SFPU files" for blackhole? Softplus is not the only one. You'll see the same for all mod, trunc, bitwise or, bitwise and, xor, etc. Ill add and test these files accordingly

**Comment by rtawfik01** [comment]
Closed most of the above tests here: [#10233](https://github.com/tenstorrent/tt-metal/pull/10233)
and have the rest of the missing sfpu functions in this issue: #10226 
will close this one
